quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/configuration/mappers/HttpPropertyMappers.java

```diff
@@ -114,6 +114,13 @@ public static PropertyMapper<?>[] getHttpPropertyMappers() {
                 fromOption(HttpOptions.HTTP_POOL_MAX_THREADS)
                         .to("quarkus.thread-pool.max-threads")
                         .paramLabel("threads")
+                        .build(),
+                fromOption(HttpOptions.HTTP_METRICS_HISTOGRAMS_ENABLED)
```

Comment:
I was wondering about the semantics of these options as to whether we should gather all metrics-related stuff into an individual option category - Metrics. I understand it's related only to HTTP, but from the UX perspective, I'd choose to separate it as the metrics options might be hidden in chunks of other options. 

I can imagine a situation when user needs/wants to use and set some settings for metrics rather in a separate manner than setting of the HTTP server. To be more specific, I'd say the most common user case is to set the necessary stuff to have Keycloak operating and then add some additional things for manageability/observability - metrics.

I know we already have the `cache-metrics-histogram-enabled` option tied to the cache options and not sure what migration would need to be done (is it already part of some release?).

TL;DR I'd opt to keep metrics-related stuff in the `METRICS` category.

Reply:
I wouldn't mind either, and the cache metrics is scheduled on KC25 so it hasn't been released, yet. 

Looking at Quarkus, they took the approach to put `metrics-enabled` at the end, see `quarkus.scheduler.metrics.enabled` or `quarkus.hibernate-orm.metrics.enabled`

Reply:
Considering it again, it'd probably make more sense to keep it as is. As you pointed to the Quarkus approach, it's quite natural for them to attach these properties to particular extensions - separated piece of work. And it make sense as we can think about these categories as the whole module as it should be delivered as one big piece. Basically, without HTTP "module", there would not be any HTTP metrics, etc.

So, +1 keep it as is.